lstchain/clean/data_clean.py

```diff
@@ -0,0 +1,40 @@
+import os
+import json
+import numpy as np
+
+__all__ = [
+        'mc_filter',
+        'data_filter'
+]
+
+file = os.path.join(os.path.dirname(__file__),"../data/data_selection_cuts.json")
+
+with open(file) as json_file:
+    data_cut = json.load(json_file)
+
+### TODO: We can generalize for the user to choose which selection cuts to make
+def mc_filter(data):
+    if 'leakage2_intensity' in data.columns:
+        data = data[data['leakage2_intensity'] < data_cut["general"]["leakage2_intensity"]]
+    else:
+        data = data[data['leakage_intensity_width_2'] < data_cut["general"]["leakage_intensity_width_2"]]
+    data = data[data['intensity'] > data_cut["general"]["intensity"]]
+    data = data[data['wl'] > data_cut["general"]["wl"]]
+    data = data[data['gh_score'] > data_cut["general"]["gh_score"]]
+    data = data[data['theta'].value < data_cut["mc"]["theta_cut"]]
+    data = data[data['source_fov_offset'].value < data_cut["general"]["source_fov_offset"]]
+
+    return data
+
+def data_filter(data):
```

Comment:
such function already exists
`lstchain.io.io.filter_events`

Reply:
You might want a more specific function for your use case but you can reuse `filter_events`

Reply:
I thought that using filter_events and applying cuts, might be slightly different, hence tried this way. But I will try with filter_events and check it.

Reply:
simple suggestion, if it does not fit your needs, make your function no problem (it would just be less code)

Reply:
Filtering events is applying cuts. No semantic difference.